tests/maximal_0.wcon

```diff
@@ -0,0 +1 @@
+{"tracker-commons":true,"files":{"this":"_0", "prev":null, "next":["_1", "_2"]},"units":{},"data":[{"id":3, "t":1.3, "x":[3,4], "y":[5.4,3]}]}
```

Comment:
I don't think units are optional in the spec.  This should fail on a strict reader.